faunadb-scala/src/main/scala/faunadb/query/package.scala

```diff
@@ -44,6 +44,8 @@ package query {
     // rather transform the value to a series of object constructions.
     implicit def quotedValue(unwrapped: Value) = Expr(ObjectV("quote" -> unwrapped))
     implicit def quotedResult(unwrapped: Result[Value]) = Expr(ObjectV("quote" -> unwrapped.get))
+
+    implicit def encode[T : Encoder](obj: T): Expr = Value.from[T](obj)
```

Comment:
No, because I made `Value.from` just return a `ObjectV`, this function actually is calling the above implicit function to wrap into an `Expr`. We actually need to fix the above function to wrap the objects with `object` instead of `quote`, unless we decide to do it completely different